spec/decorators/representative_decorator_spec.rb

```diff
@@ -0,0 +1,4 @@
+require 'spec_helper'
+
+describe RepresentativeDecorator do
```

Comment:
Sei como é. Testes parecem ser inúteis e trabalho a mais que não vale pra nada (pelo menos parecia pra mim assim no começo). A gente só sabe o quanto eles podem nos ensinar a fazer as coisas quando a gente começa a implementar primeiro o teste e depois a funcionalidade.